accepted/0000-npm-pkg.md

```diff
@@ -0,0 +1,93 @@
+# npm pkg
+
+## Summary
+
+A new top-level command that manages any property in your `package.json`.
+
+## Motivation
+
+Some users prefer managing their `package.json` using **npm cli** comands rather than manually editing a JSON file.
+
+As of today, some of the **npm cli** commands already handle the automated management of the `package.json` file, such as `npm install` adding / updating dependency declarations, `npm version` bumping the version value, or in the case of `npm set-script` a top-level command that serves specifically the purpose of managing `"scripts"` data in the `package.json` file.
+
+## Detailed Explanation
+
+- Add a `npm pkg` command that has the following subcommands:
+  - `npm pkg get <value>`
+  - `npm pkg set <key>=<value>`
+  - `npm pkg delete <key>`
+
+## Rationale
+
+Making manage your `package.json` file something less manually curated and more of an automated experience.
+
+## Implementation
+
+- Add a new `lib/pkg.js` top-level command that will handle adding / removing / modifying properties in your `package.json` file.
+- Reuses the same logic existing to query for object properties from `lib/view.js`
+- Throws when trying to replace current literal values with object notations
+- Type cast via an extra cli option: `--pkg-type=[number|boolean]`
+
+### Example
+
+```
+$ npm pkg set scripts.lint=eslint
+$ npm pkg get scripts
+{
+  "lint": "eslint"
+}
+$ npm pkg get scripts.lint
+"eslint"
+$ npm pkg set scripts.lint.eslint.foo=bar
+error: "scripts.lint" exists and is not an object
+$ npm pkg delete scripts
+$ npm pkg set scripts.lint.eslint.foo=bar
```

Comment:
any reason this couldn't be combined into, like, `npm pkg set -f scripts.lint.eslint.foo=bar`?

Reply:
I like that idea! 😄